http/util/src/main/java/org/wildfly/security/http/util/SetRequestInformationCallbackMechanismFactory.java

```diff
@@ -31,7 +27,11 @@
 import java.util.Map;
 import java.util.function.Function;
```

Comment:
Could you also reorder the java and javax ones so they follow the order mentioned on the Jira issue?
Thanks